### ISSUE TITLE
fzf_action.files does not work when -layout=floating is specified

### ISSUE THREAD

**Issue by yanma** [obaland/vfiler-fzf#1]
Hello, I'm quite interested in this plugin but found that fzf_action.files does not work when `-layout=floating` is specified...
The minimum `.vimrc` which reproduces the phenomenon is below.

```vimrc
" ======== vim-plug
if filereadable(expand("$HOME/.vim/autoload/plug.vim"))
    call plug#begin('~/.vim/plugged')
    Plug 'junegunn/fzf', {'do': {-> fzf#install()}}
    Plug 'junegunn/fzf.vim'
    Plug 'obaland/vfiler.vim'
    Plug 'obaland/vfiler-fzf'
    call plug#end()
endif

" ======== VFiler
lua << EOF
local fzf_action = require('vfiler/fzf/action')
require'vfiler/config'.setup {
    mappings = {
        ['f'] = fzf_action.files,
        },
    }
EOF
```

In this environment, I could trigger fzf by using `:VFiler -layout=floating`. But after I selected a file, it outputs error message as below. If the layout is not floating, it works perfectly.

```
Error detected while processing function 4[30]..<SNR>13_callback:
line   23:
Vim(call):E5108: Error executing lua ...iroaki/.vim/plugged/vfiler-fzf/lua/vfiler/fzf/action.lua:79: attempt to index local 'current' (a nil value)^@stack traceback:^@^I...iroaki/.vim/plugged/vfiler-fzf/lua/vfiler/fzf/action.lua:79: in function '_sink'^@^I[string "luaeval()"]:1: in main chunk
Press ENTER or type command to continue
```

My nvim version is v0.7.2.

```bash
$ nvim --version
NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

If you need further information, please feel free to ask me. Thank you for developing this useful software!

**Comment by obaland** [comment]
Thank you for your detailed report.
Also apologize for any inconvenience caused.

Please give us some time to investigate the cause.

**Comment by obaland** [comment]
Sorry for the delay. Status report.
I have found the cause of the problem, but we are still looking for a better way to correct it.
Due to our time constraints, I would appreciate it if you could give me some time to resolve this issue.

**Comment by obaland** [comment]
@yanma 
I have corrected this in e5659281fa911a341516c8f86fe62e011fd87050.
The [vfiler](https://github.com/obaland/vfiler.vim/commit/774051265cce07f1025217afedc9cfe8455d8f4b) itself has also been corrected. Please check the latest version of each.

**Comment by yanma** [comment]
Thanks a lot for your hard effort!
Now it works perfectly even when I use floating layout.
I like this extension.
